packages/stylelint-config/index.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
 			},
 		],
 		'custom-property-pattern': [
-			'^([a-z][a-z0-9]*)(-[a-z0-9]+)*$|^wp--([a-z][a-z0-9]*)([-]{1,2}[a-z0-9]+)*$',
+			'^([a-z][a-z0-9]*)(-[a-z0-9]+)*$|^wp[-]{1,2}([a-z][a-z0-9]*)([-]{1,2}[a-z0-9]+)*$'
```

Comment:
You removed the comma at the end thus this is invalid code and don't even pass linting :).

Reply:
🤦 fixed